gnomad/utils/vcf.py

```diff
@@ -207,11 +205,51 @@
         "Number": "A",
         "Description": "Maximum p-value over callset for binomial test of observed allele balance for a heterozygous genotype, given expectation of 0.5",
     },
+    "monoallelic": {
+        "Description": "All samples are homozygous alternate for the variant"
+    },
+    "QUALapprox": {
+        "Number": "1",
+        "Description": "Sum of PL[0] values; used to approximate the QUAL score",
+    },
+    "AS_SB_TABLE": {
+        "Number": ".",
+        "Description": "Allele-specific forward/reverse read counts for strand bias tests",
+    },
 }
 """
 Dictionary used during VCF export to export row (variant) annotations.
 """
 
+IN_SILICO_ANNOTATIONS_INFO_DICT = {
+    "cadd_raw_score": {
+        "Number": "1",
+        "Description": "Raw CADD scores are interpretable as the extent to which the annotation profile for a given variant suggests that the variant is likely to be 'observed' (negative values) vs 'simulated' (positive values). Larger values are more deleterious.",
+    },
+    "cadd_phred": {
+        "Number": "1",
+        "Description": "Cadd Phred-like scores ('scaled C-scores') ranging from 1 to 99, based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Larger values are more deleterious",
```

Comment:
```suggestion
        "Description": "Cadd Phred-like scores ('scaled C-scores') ranging from 1 to 99, based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Larger values are more deleterious.",
```
^ the other in silico descriptions have periods at the end